README.md

```diff
@@ -45,20 +45,31 @@ Supported operating system versions:
 
 * CentOS 7
 * Debian 7.11
-* Debian 8.6
-* FreeBSD 10.1
+* Debian 8.9
+* (Debian 9.1)
 * FreeBSD 10.3
+* FreeBSD 11
 * Ubuntu 14.04
 * Ubuntu 16.04
 
+_Currently, Debian 9 is not supported due to a known bug in Zonemaster::LDNS. That has to be resolved 
+before we can include Debian 9 as supported OS._
+
 Supported database engine versions:
 
 * MySQL 5.5, 5.7
 * PostgreSQL 9.3, 9.4, 9.5
```

Comment:
Maybe update the supported databases too? I went ahead and compiled the eligible ones. Maybe we should ask Michal?

MySQL 5.5 (Debian 7.11, 8.9, FreeBSD, Ubuntu 14.04)
MySQL 5.6 (FreeBSD)
MySQL 5.7 (FreeBSD, Ubuntu 16.04)
MySQL 8.0 (FreeBSD)
MariaDB 5.5 (Centos 7) ?
Postgres 9.1 (Debian 7.11)
Postgres 9.2 (Centos 7, FreeBSD)
Postgres 9.3 (FreeBSD, Ubuntu 14.04)
Postgres 9.4 (Debian 8.9, FreeBSD)
Postgres 9.5 (FreeBSD, Ubuntu 16.04)
Postgres 9.6 (FreeBSD)
Postgres 10.0 (FreeBSD)

Reply:
We can only support MariaDB if it is really a plug-in replace for MySQL. At least an update of the installation documentation is needed.

Should we really support all those versions on FreeBSD? Maybe only MySQL 8.0 and PostgreSQL 10? Could be extended to 8.0+5.7 and 10.0+9.6, repectively?